app/app.go

```diff
@@ -0,0 +1,9 @@
+package app
+
+import (
+	"go-capstone/routes"
+)
+
+func Run() {
```

Comment:
By convention, all the exported stuff (function, variable, struct, etc) should have description
```go
// Run - Description here
func Run() {
```